RoboFile.php

```diff
@@ -351,6 +351,53 @@ function qaCodeSniffer($severity='errors') {
       ->run();
   }
 
+  function qaFixFile($filePath) {
+    if (substr($filePath, -4) === '.php') {
+      // fix PHPCS rules
+      return $this->collectionBuilder()
+        ->taskExec(
+          './tasks/code_sniffer/vendor/bin/phpcbf ' .
+            '--standard=./tasks/code_sniffer/MailPoet ' .
+            '--runtime-set testVersion 5.6-7.3 ' .
+            $filePath . ' -n'
+        )
+        ->run();
+    }
+    if (substr($filePath, -4) === '.jsx') {
+      // fix ESLint using ES6 rules
+      return $this->collectionBuilder()
+        ->taskExec(
+          'npx eslint -c .eslintrc.es6.json ' .
+            '--max-warnings 0 ' .
+            '--fix ' .
+            $filePath
+        )
+        ->run();
+    }
+    if (substr($filePath, -3) === '.js') {
+      // fix ESLint using ES5 rules
+      return $this->collectionBuilder()
+        ->taskExec(
+          'npx eslint -c .eslintrc.es5.json ' .
+            '--max-warnings 0 ' .
+            '--fix ' .
+            $filePath
+        )
+        ->run();
+    }
+    if (substr($filePath, -8) === '.spec.js') {
```

Comment:
This will be first run through the previous IF since it also ends with `.js`. I suggest using `elseif` and changing the order.

Reply:
I need to change the order but no need to use `elseif` because I am doing `return`.